### PR TITLE
[WIP] Multiline function call #111 test

### PR DESCRIPTION
**Pull request by SpacePossum**
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/111

### CODE REVIEW

File: Symfony/CS/Tests/Fixer/PSR2/MethodArgumentSpaceFixerTest.php [hunk 1]
```diff
@@ -29,6 +29,20 @@ public function testFix($expected, $input = null)
...
+            array(
+'<?php
+    $e = new \\Exception(
+        \'test\',
+        100);
+    throw $e;',
+'<?php
+    $e = new \\Exception(\'test\',
+                         100);
+    throw $e;'),
```
Comment (keradus) [inline Symfony/CS/Tests/Fixer/PSR2/MethodArgumentSpaceFixerTest.php]: `)` should be in new line

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/PSR2/MethodArgumentSpaceFixerTest.php]: fabbot / php-cs-fixer like
```php
    throw $e;', ),
```

I'll try the new line first

Comment (keradus) [inline Symfony/CS/Tests/Fixer/PSR2/MethodArgumentSpaceFixerTest.php]: You open `(`, then start an element of array at next line, then you should put `)` on separate line just like with `(`.
And then - we just like to have commas after each element of multiline arrays ;)

**Comment by keradus** [comment]
```
* **method_argument_space** [PSR-2]
                In method arguments and method call,
                there MUST NOT be a space before each
                comma and there MUST be one space
                after each comma.
```

I do not agree that the first added test is bug of described fixer. This fixer does not move params to new lines.
Multiline function calls are defined by other PSR rule and therefore should be done as separate fixer.
Do you agree?

**Comment by SpacePossum** [comment]
This issue could be resolved by another fixer, but until there is such a fixer I think the test added is a relevant issue that should be addressed by the PHP-CS-Fixer.
Feel free to close this PR, I don't think I can work on this anytime soon :(

**Comment by keradus** [comment]
OK, then this is not a bug report but feature request.
It's cool feature, but it is already requested by #111 . It's great PR and yeah, I would love to see it.
Closing as duplicate.

Thank you @SpacePossum

**Pull request closed without merging**